src/Symfony/Component/Messenger/Tests/Command/FailedMessagesRemoveCommandTest.php

```diff
@@ -19,6 +19,23 @@
 
 class FailedMessagesRemoveCommandTest extends TestCase
 {
+    public function testRemoveSingleMessage()
+    {
+        $receiver = $this->createMock(ListableReceiverInterface::class);
+        $receiver->expects($this->once())->method('find')->with(20)->willReturn(new Envelope(new \stdClass()));
+
+        $command = new FailedMessagesRemoveCommand(
+            'failure_receiver',
+            $receiver
+        );
+
+        $tester = new CommandTester($command);
+        $tester->execute(['id' => 20, '--force' => true]);
```

Comment:
This looks like a wrong execution of the command to me as the argument is declared as an array argument. Does this error only occur when you execute the command programmatically?

Reply:
Yes, it does. But these are reasons why I think it is a bug:
* I worked for me before, at least at versions 5.0.x
* The name `id` and description `Specific message id(s) to remove` of the input argument suggest that I can use single id, but I have to use multiple (array) of ids
* There is a test case for it in 5.0.x (what I am adding in, again)